### PR TITLE
custom markets

### PR DESCRIPTION
**Pull request by CleanBread**

### CODE REVIEW

File: pages/hub/apps/guides/custom-markets.mdx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import { getMarketName, getSelectionName } from '@azuro-org/dictionaries'
+
+conditions.forEach((condition) => {
+  const customMarketName = condition.title
+
+  condition.outcomes.forEach((outcome) => {
+    const customSelectionName = outcome.title
+
+    // If customMarketName has a value, then use it as marketName; otherwise, use the getMarketName function.
+    const marketName = customMarketName || getMarketName({ outcome.outcomeId })
```
Comment (on47sky) [inline pages/hub/apps/guides/custom-markets.mdx:45]: please fix param in getMarketName and getSelectionName

**Review by on47sky — APPROVED** [review]

File: pages/hub/apps/guides/custom-markets.mdx [hunk 2]
```diff
@@ -0,0 +1,51 @@
+import { Callout } from 'components'
+
+The Azuro Protocol also offers markets for world events, such as the "2024 Presidential Election USA."
```
Comment (Avvrik) [inline pages/hub/apps/guides/custom-markets.mdx:3]: Apart from the predominant sports markets, Azuro now allows to integrate political and social markets, starting with the "2024 Presidential Election USA" market.

File: pages/hub/apps/guides/custom-markets.mdx [hunk 3]
```diff
@@ -0,0 +1,51 @@
+import { Callout } from 'components'
+
+The Azuro Protocol also offers markets for world events, such as the "2024 Presidential Election USA." 
+
+<Callout type="info">
+  If you use our [SDK](/hub/apps/sdk/overview), no additional changes are needed; just ensure you are using version 5.1.0 or above.
+</Callout>
+
+The steps to obtain custom markets are the same as for sports markets, but there is a slight difference in how the names of markets and selections are generated.
```
Comment (Avvrik) [inline pages/hub/apps/guides/custom-markets.mdx:9]: The steps to get custom (i.e. markets other than sport markets) are the same....

**Review by Avvrik — APPROVED** [review]

File: pages/hub/apps/guides/custom-markets.mdx (hunk 2 above)
Comment (CleanBread) [reply pages/hub/apps/guides/custom-markets.mdx]: ed713387326b0f5d2cc3aa32c1ea95d8f69953b4

File: pages/hub/apps/guides/custom-markets.mdx (hunk 3 above)
Comment (CleanBread) [reply pages/hub/apps/guides/custom-markets.mdx]: ed713387326b0f5d2cc3aa32c1ea95d8f69953b4

**Pull request merged**